manifest/dynamic_manifest.go

```diff
@@ -16,7 +16,7 @@ import (
 	"golang.org/x/sync/errgroup"
 )
 
-var log = logging.Logger("f3-dynamic-manifest")
+var log = logging.Logger("f3/dynamic-manifest")
```

Comment:
Should we change it everywhere to use `/` instead of `-`? Can you have a quick look?